src/store/card.js

```diff
@@ -30,13 +30,75 @@ export default {
 		cards: [],
 	},
 	getters: {
-		cardsByStack: (state, getters) => (id) => {
-			return state.cards.filter((card) => card.stackId === id && (getters.getSearchQuery === '' || (card.title.toLowerCase().includes(getters.getSearchQuery.toLowerCase()) || card.description.toLowerCase().includes(getters.getSearchQuery.toLowerCase())))
-			).sort((a, b) => a.order - b.order)
-		},
-		cardById: state => (id) => {
-			return state.cards.find((card) => card.id === id)
-		},
+		cardsByStack: (state, getters, rootState) => (id) => {
+			return state.cards
+				.filter((card) => card.stackId === id && (getters.getSearchQuery === '' || (card.title.toLowerCase().includes(getters.getSearchQuery.toLowerCase()) || card.description.toLowerCase().includes(getters.getSearchQuery.toLowerCase())))
```

Comment:
We should move this filter rule below the other one, so that the string filter operations are performed on the filtered subset already.